modules/tinymce/src/core/main/ts/newline/InsertBlock.ts

```diff
@@ -249,7 +250,7 @@ const insert = (editor: Editor, evt?: EditorEvent<KeyboardEvent>): void => {
   const createNewBlock = (name?: string): Element => {
     let node: Node | null = container;
     const textInlineElements = schema.getTextInlineElements();
-
```

Comment:
This blank separator line should not have been removed.

Reply:
My mistake